include/mbedtls/ecdsa.h

```diff
@@ -255,6 +287,33 @@ int mbedtls_ecdsa_write_signature_det( mbedtls_ecdsa_context *ctx,
 #endif /* MBEDTLS_DEPRECATED_REMOVED */
 #endif /* MBEDTLS_ECDSA_DETERMINISTIC */
 
+/**
+ * \brief           Convert an ECDSA signature from number pair format to ASN.1
+ *
+ * \param r         First number of the signature
+ * \param s         Second number of the signature
+ * \param sig       Buffer that will hold the signature
+ * \param slen      Length of the signature written
+ * \param ssize     Size of the sig buffer
+ *
+ * \note            The size of the buffer \c ssize should be at least
+ *                  `MBEDTLS_ECDSA_MAX_SIG_LEN(grp->pbits)` bytes long if
+ *                  the signature was produced from curve \c grp,
+ *                  otherwise this function will return an error.
+ *                  The output ASN.1 SEQUENCE format is as follows:
+ *                  Ecdsa-Sig-Value ::= SEQUENCE {
+ *                              r       INTEGER,
+ *                              s       INTEGER
+ *                          }
+ *
```

Comment:
The documentation of this function should note that the output is the format expected by `mbedtls_ecdsa_verify`.

(Preexisting) The documentation of `mbedtls_ecdsa_sign` and `mbedtls_ecdsa_verify` should specify the format of the produced/expected signature.

The format is from SEC1 (http://www.secg.org/SEC1-Ver-1.0.pdf), also described in RFC 5480.

Reply:
Done